quipucords/tests/integration/test_openshift_scan.py

```diff
@@ -48,22 +50,20 @@ def node_resources():
 @pytest.fixture
 def expected_node(node_resources):
     """Return a list with OCP node."""
-    return [
-        OCPNode(
-            name="node name",
-            errors={},
-            labels={"node-role.kubernetes.io/master": ""},
-            taints=[{"key": "some", "effect": "some"}],
-            capacity=node_resources,
-            addresses=[{"type": "ip", "address": "1.2.3.4"}],
-            machine_id="<MACHINE-ID>",
-            allocatable=node_resources,
-            architecture="amd64",
-            kernel_version="4.18.0-305.65.1.el8_4.x86_64",
-            operating_system="linux",
-            creation_timestamp="2022-12-18T03:56:20Z",
-        )
-    ]
+    return OCPNode(
+        name="node name",
+        errors={},
+        labels={"node-role.kubernetes.io/master": ""},
+        taints=[{"key": "some", "effect": "some"}],
+        capacity=node_resources,
+        addresses=[{"type": "ip", "address": "1.2.3.4"}],
+        machine_id="<MACHINE-ID>",
+        allocatable=node_resources,
+        architecture="amd64",
+        kernel_version="4.18.0-305.65.1.el8_4.x86_64",
+        operating_system="linux",
+        creation_timestamp="2022-12-18T03:56:20Z",
+    )
```

Comment:
since this fixture is `expected_node` (singular), I changed it from a list of nodes to just one node